crashes.json

```
@@ -1055,6 +1116,231 @@
+      "causes": [
+        {
+          "method": "contains",
+          "value": "tryrite.brite"
+        }
+      ]
+    },
+    {
+      "fix": "AwayFromAuction discovered. Consider using NotEnoughUpdate's `/neuah` instead (`-mod neu`) or buy Booster Cookies from the Bazaar in-game.",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "awayfromauction"
+        }
+      ]
+    },
+    {
+      "fix": "Salvage Assister is in your mods folder. Consider using the equivalent feature from Skytils (`-mod skytils`) or Danker's Skyblock Mod (`-mod dsm`) instead.",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "salvageassister"
+        }
+      ]
+    },
+    {
+      "fix": "FarmingUtils found. Consider using the equivalent feature `Skill Overlays —> Farming` from NotEnoughUpdates (`-mod neu`).",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "farmingutils"
+        }
+      ]
+    },
+    {
+      "fix": "BedTime was found. Consider using the equivalent feature `/neuah` from NotEnoughUpdates (`-mod neu`).",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "bedtime"
+        }
+      ]
+    },
+    {
+      "fix": "GoldenEnchant mod unearthed. Consider using the equivalent feature `/neuec` from NotEnoughUpdates (`-mod neu`).",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "me.GoldenEnchant"
+        }
+      ]
+    },
+    {
+      "fix": "timestampmod mod unearthed. Consider using equivalent features from Cowlection (`-mod cowlection`) and SkyblockAddons (`-mod sba`).",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "anna.timestamp"
+        }
+      ]
+    },
+    {
+      "fix": "reswitch detected. Use NEU's `Dwarven Mines Textures` feature instead, as all necessary Dwarven Mines block textures provided by your resource packs are already loaded on startup this way.",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "resourceswitch"
+        }
+      ]
+    },
+    {
+      "fix": "\"near\" detected. On the fence about Skyclient, hm?",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "studio.dreamys."
+        }
+      ]
+    },
+    {
+      "fix": "\"BetterSkyblockMod\" was found. It has extensive overlap with many features already present in `-mods`, including Skytils, NotEnoughUpdates, and Chatting.",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "darthvader.dbsm."
+        }
+      ]
+    },
+    {
+      "fix": "\"SkyblockChatFilter\" has been detected. Remove it and set Chatting's \"Spam Blocker\" threshold to 95 instead (`-mod chatting`).",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "taj_ny.skyblockchatfilter"
+        }
+      ]
+    },
+    {
```

```suggestion
    },
    {
```
i hate to do this to my own feature but sadly @Wyvest is too lazy to make a release that includes the spam blocker